package-lock.json

```diff
@@ -44,10 +82,32 @@
         "negotiator": "0.6.2"
       }
     },
+    "acorn": {
+      "version": "7.4.0",
+      "resolved": "https://registry.npmjs.org/acorn/-/acorn-7.4.0.tgz",
+      "integrity": "sha512-+G7P8jJmCHr+S+cLfQxygbWhXy+8YTVGzAkpEbcLo2mLoL7tij/VG41QSHACSf5QgYRhMZYHuNc6drJaO0Da+w==",
+      "dev": true
+    },
+    "acorn-jsx": {
```

Comment:
@vegetabill - should this jsx be removed? This was generated automatically when I ran `rm package-lock.json && rm -rf node_modules/ && npm i` after removing react and jsx as part of package.json. Guessing we can leave it as-is.

Reply:
Yes it must be an indirect dependency. We should just leave it then. Thanks for checking!